RUFAS/biophysical/manure/manure_manager.py

```diff
@@ -692,6 +697,10 @@ def _create_all_processors(
             if not (issubclass(processor_initializer, Handler) or issubclass(processor_initializer, Separator)):
                 del processor_config["processor_type"]
             processor = processor_initializer(**processor_config)
+            if isinstance(processor, AnaerobicLagoon) or isinstance(processor, SlurryStorageOutdoor):
+                processor.intercept_mean_temp = intercept_mean_temp
+                processor.phase_shift = phase_shift
+                processor.amplitude = amplitude
```

Comment:
```suggestion
            if isinstance(processor, (AnaerobicLagoon, SlurryStorageOutdoor)):
                processor.intercept_mean_temp = intercept_mean_temp
                processor.phase_shift = phase_shift
                processor.amplitude = amplitude
```
It looks like `isinstance` can take a single class or a tuple.